### PR TITLE
Convert more links to use new routes

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: src/realm-settings/RealmSettingsSection.tsx [hunk 1]
```diff
@@ -52,7 +53,11 @@ export const EditProviderCrumb = () => {
   return (
     <>
       <Breadcrumb>
-        <BreadcrumbItem to={`#/${realm}/realm-settings/keys`}>
+        <BreadcrumbItem
+          component={Link}
+          // @ts-ignore
+          to={toRealmSettings({ realm, tab: "keys" })}
```
Comment (jenny-s51) [inline src/realm-settings/RealmSettingsSection.tsx:59]: This should redirect back to `/realm-settings/keys`

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Nice work @jonkoops - most of these paths work great! One small issue with the breadcrumb but otherwise LGTM. Thank you for putting up your routing PRs in medium-ish chunks like this -- it's helpful for reviewing :)

File: src/realm-settings/RealmSettingsSection.tsx (hunk 1 above)
Comment (jonkoops) [reply src/realm-settings/RealmSettingsSection.tsx:59]: Good find! This seems to be a bug in Patternfly, I've created a PR to get it fixed: https://github.com/patternfly/patternfly-react/pull/6111

Comment (jonkoops) [reply src/realm-settings/RealmSettingsSection.tsx]: I've fixed this by using the `render` prop instead.

**Comment by ssilvert** [comment]
@jenny-s51 We are merging this.  Please let us know if you have any issues.

**Pull request merged**